pay-api/src/pay_api/__init__.py

```diff
@@ -90,3 +91,13 @@ def shell_context():
         return {'app': app, 'jwt': jwt, 'db': db, 'models': models}  # pragma: no cover
 
     app.shell_context_processor(shell_context)
+
+
+def build_cache(app):
```

Comment:
may be make it as private?

Reply:
Yes, will change in next PR if that's ok.